rotkehlchen/db/addressbook.py

```diff
@@ -94,10 +94,11 @@ def add_addressbook_entries(
                         (entry.address, ANY_BLOCKCHAIN_ADDRESSBOOK_VALUE),
                     )
 
-                write_cursor.execute(
-                    'INSERT INTO address_book (address, name, blockchain) VALUES (?, ?, ?)',
-                    entry.serialize_for_db(),
-                )
+                if entry.name != '':  # accounts with no name should not have an addressbook entry
```

Comment:
i don't think this is the best approach. looking at the type of `Addressbook` and where it's used(schemas, etc.), i don't think empty strings are allowed.

Reply:
As discussed in discord, this has been updated to use a different approach that doesn't involve using the functions from addressbook.